bundles/org.eclipse.passage.lbc.api/src/org/eclipse/passage/lbc/internal/api/BackendConditionDispatcher.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lbc.internal.api;
+
+import org.eclipse.passage.lic.internal.api.conditions.ConditionPack;
+
+/**
+ * @since 1.0
+ */
+public interface BackendConditionDispatcher {
```

Comment:
I would assume you don't have "condition pack" yet but have a "request" for it
And this request is something sufficient to understand 
1) who is requesting
2) what is requested

And then you need to give a response that contains a "condition pack" or something similar.
Finally your response should be understood by `ConditionTransport` on the client side.